appkit/src/test/scala/org/ergoplatform/appkit/ErgoAuthSpec.scala

```diff
@@ -0,0 +1,41 @@
+package org.ergoplatform.appkit
+
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+import scorex.util.Random
+import sigmastate.interpreter.HintsBag
+
+import java.nio.charset.StandardCharsets
+
+class ErgoAuthSpec extends PropSpec with Matchers with ScalaCheckDrivenPropertyChecks
+  with AppkitTestingCommon {
+
+  property("ErgoAuth address roundtrip") {
+    // server side
+    val serializedSigmaBoolean = ErgoAuthUtils.serializeSigmaBoolean(address)
+    // message to sign should be something random and not repeating
+    val requestedMessage = addrStr + System.currentTimeMillis().toString
+
+    // transferred to client, and now we are on client side
+    val signedMessage = new String(Random.randomBytes(16)) + requestedMessage +
+      new String(Random.randomBytes(32))
```

Comment:
What these random strings represent?

Reply:
It is this step from the EIP:

> 6) When the user agrees, the wallet app **adds some own bytes** to the obtained message from ErgoAuthRequest,
> signs it  and sends the signed message to the ErgoAuthRequest's replyToUrl.

Reply:
I suggest to add this discription to the code comment (in test)

Reply:
done